big-practice/src/style/components/button.css

```diff
@@ -1,5 +1,6 @@
 /* START BUTTON */
 .primary-btn {
+  display: inline-block;
```

Comment:
Why don't you set all common style in .btn class?

Reply:
fixed